### ISSUE TITLE
resolve EC:4.4.1.35, EC:4.4.1.1, EC:4.4.1.28 mappings

### ISSUE THREAD

**Issue by balhoff** [geneontology/go-ontology#27861]
![0c780d40643465c6e821554b78c892c481e811457177dd4e0645a52540f62fb2](https://github.com/geneontology/go-ontology/assets/210210/433f3698-95fb-4640-89a9-1d580478673e)

- [H2O + L-cysteine = H(+) + hydrogen sulfide + NH4(+) + pyruvate](http://rdf.rhea-db.org/24931) EquivalentTo [EC:4.4.1.28](http://purl.uniprot.org/enzyme/4.4.1.28)	(most probable)	0.75
- [L-cystine L-cysteine-lyase (deaminating)](http://purl.obolibrary.org/obo/GO_0044540) EquivalentTo [H2O + L-cystine = NH4(+) + pyruvate + S-sulfanyl-L-cysteine](http://rdf.rhea-db.org/24927)	(most probable)	0.8
- [L-homoserine = 2-oxobutanoate + NH4(+)](http://rdf.rhea-db.org/24923) ProperSubClassOf [EC:4.4.1.1](http://purl.uniprot.org/enzyme/4.4.1.1)		0.1
- [L-cysteine desulfhydrase activity](http://purl.obolibrary.org/obo/GO_0080146) ProperSubClassOf [EC:4.4.1.1](http://purl.uniprot.org/enzyme/4.4.1.1)		0.08
- [L-cysteine desulfhydrase activity](http://purl.obolibrary.org/obo/GO_0080146) EquivalentTo [H2O + L-cysteine = H(+) + hydrogen sulfide + NH4(+) + pyruvate](http://rdf.rhea-db.org/24931)	(most probable)	0.8
- [L-cystine L-cysteine-lyase (deaminating)](http://purl.obolibrary.org/obo/GO_0044540) ProperSubClassOf [EC:4.4.1.1](http://purl.uniprot.org/enzyme/4.4.1.1)		0.08
- [cystathionine gamma-lyase activity](http://purl.obolibrary.org/obo/GO_0004123) EquivalentTo [H2O + L,L-cystathionine = 2-oxobutanoate + L-cysteine + NH4(+)](http://rdf.rhea-db.org/14005)	(most probable)	0.8
- [H2O + L,L-cystathionine = 2-oxobutanoate + L-cysteine + NH4(+)](http://rdf.rhea-db.org/14005) EquivalentTo [EC:4.4.1.1](http://purl.uniprot.org/enzyme/4.4.1.1)	(most probable)	0.75
- [cystathionine gamma-lyase activity](http://purl.obolibrary.org/obo/GO_0004123) EquivalentTo [EC:4.4.1.1](http://purl.uniprot.org/enzyme/4.4.1.1)	(most probable)	0.8
- [H2O + L-cysteine = H(+) + hydrogen sulfide + NH4(+) + pyruvate](http://rdf.rhea-db.org/24931) ProperSubClassOf [EC:4.4.1.1](http://purl.uniprot.org/enzyme/4.4.1.1)		0.1
- [H2O + L-cystine = NH4(+) + pyruvate + S-sulfanyl-L-cysteine](http://rdf.rhea-db.org/24927) EquivalentTo [EC:4.4.1.35](http://purl.uniprot.org/enzyme/4.4.1.35)	(most probable)	0.75

**Comment by pgaudet** [comment]
@balhoff will re-run boomer

**Comment by balhoff** [comment]
This no longer seems to be an issue with the most recent boomer run.

**Comment by sjm41** [comment]
Yep, looks like it was one of the things addressed in https://github.com/geneontology/go-ontology/issues/28011
